Config.php

```diff
@@ -186,6 +187,11 @@ public static function shouldAppendUserEmailSuffixToUsername()
         return self::getConfigOption('append_user_email_suffix_to_username') == 1;
     }
 
+    public static function getStripDomainFromWebAuth()
+    {
+        return self::getConfigOption('strip_domain_from_web_auth');
```

Comment:
Can we model this like the other functions that return bools? Ie, `getConfigOption(...) == 1`?

Reply:
Sure thing, 'twas but an oversight.